### ISSUE TITLE
Flattening Criterion

### ISSUE THREAD

**Issue by luckyboy101** [Jhonve/ImageSmoothing#1]
# flat_loss
output_copy = output.clone()
flat_output = gaussian_kernel(output_copy)
flat_map = (edge_map - 1.) * -1.
loss_flat = F.mse_loss(flat_map * output, flat_map * flat_output)
这部分和原文的实现不一样，训练出来的效果不会好，对吧？

**Comment by Jhonve** [comment]
Yes. Actually, this repository is not a reproduction of paper 'Image Smoothing via Unsupervised Learning, Fan et al. ACM TOG (SIGGRAPH ASIA 2018)' (this repository is only my homework).  Sorry for misleading and I will update the README.md later.
是的，和那篇siggraph asia的文章有区别，这里只是使用控制边界的方式做了一次尝试，效果确实没有原文的好。原文中的方法需要使用CUDA编写一个Operator，由于时间原因，没有来得及实现，后续有时间会继续更新。感谢您的关注！
